plugins/chain_plugin/test/test_tracked_storage.cpp

```diff
@@ -0,0 +1,194 @@
+#define BOOST_TEST_MODULE tracked_storage
+#include <boost/test/included/unit_test.hpp>
+#include <eosio/chain_plugin/tracked_storage.hpp>
+#include <sstream>
+
+using boost::multi_index_container;
+using namespace boost::multi_index;
+using namespace eosio::chain_apis;
+
```

Comment:
Same test that is in fc, right? Should remove from here.

Reply:
cleaned up now